docs/source-pytorch/accelerators/tpu_basic.rst

```diff
@@ -32,26 +32,6 @@ some subset of those 2048 cores.
 
 ----
 
-Run on 1 TPU core
```

Comment:
These sections are still relevant. It is still 100% ok to set these arguments, even if they default to auto now.